terra-jupyter-bioconductor/Dockerfile

```diff
@@ -1,10 +1 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.1.5
-
-USER root
-
-## Install Bioconductor Core packages
-RUN curl -O https://raw.githubusercontent.com/anvilproject/anvil-docker/master/anvil-rstudio-bioconductor/install.R \
-	&& R -f install.R \
-	&& rm -rf install.R
-
-USER $USER
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:2.1.6
```

Comment:
@nturaga would like to run these changes by you

Reply:
This looks good to me. 

It seems the changes that were made are moving the contents of the `install.R` file into the Dockerfile explicitly to make the `terra-jupyter-r` image the same as the `terra-jupyter-bioconductor` image. 

Correct me if i'm wrong about this.

Reply:
You are correct. Part of the story is we wanted the AnVIL packages in the R image, but it just to happened to make the bioconductor image identical to the terra-jupyter-r image.